### DISCUSSION TITLE
RelationManagers in Wizard Step

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#7573] [Help]
Is it posiible to add a RelationManagers inside a Wizard Step; i created a ProductResource and im using Wizard Steps to create a record, theres a relationship ProductColors and i need to place the RelationManagers\ProductColorsRelationManager::class in the las step of the Form.

Thx in adavnced :)

**Comment by pboivin** [comment]
Unfortunately no, it's not possible to embed a relation manager into a form.
